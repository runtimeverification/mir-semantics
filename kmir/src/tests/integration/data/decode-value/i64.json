{
  "bytes": [
    241,
    255,
    255,
    255,
    255,
    255,
    255,
    255
  ],
  "types": [],
  "typeInfo": {
    "PrimitiveType": {
      "Int": "I64"
    }
  }
}
