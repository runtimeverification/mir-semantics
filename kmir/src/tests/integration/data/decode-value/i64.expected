Integer ( -15 , 64 , true )
